### PR TITLE
Drop useless files

### PR DESCRIPTION
**Pull request by MateoLostanlen**
The aim of this PR is to do a bit of cleaning:

1. we are not using pizeros so pizeros related files are removed
2. We are not usiing a wifi routeur so routeur related files / elements are removed too
3. Drop useless one test

### CODE REVIEW

**Comment by MateoLostanlen** [comment]
I will improve coverage when #83 will be merge

File: pyroengine/pi_utils/check_internet_connection.sh [hunk 1]
```diff
@@ -36,7 +36,5 @@ then
     exit 0;
 fi;
 
-python3 /home/pi/pyro-engine/pyroengine/pi_utils/reboot_router;
-sleep 60;
```
Comment (frgfm) [inline pyroengine/pi_utils/check_internet_connection.sh:40]: no longer relevant?

**Review by frgfm — COMMENTED** [review]
Thanks! I wasn't sure about the router reboot check though

File: pyroengine/pi_utils/check_internet_connection.sh (hunk 1 above)
Comment (MateoLostanlen) [reply pyroengine/pi_utils/check_internet_connection.sh:40]: no this was to reboot the wifi router that we are not using anymore

**Review by frgfm — APPROVED** [review]
Thanks!

**Pull request merged**